packages/manager-react-components/src/components/navigation/menus/guide/guide.component.tsx

```diff
@@ -66,6 +67,7 @@ export const GuideButton: React.FC<GuideButtonProps> = ({ items }) => {
               rel={item.rel}
               type={LinkType.external}
               label={item.label}
+              onClickReturn={item.onClick}
```

Comment:
**nitpick(blocker):** `onClick` is optional. So if it's not sent, this breaks.

Reply:
@anooparveti 
`onClickReturn` is also optional in the `<Links>` component (that uses `OsdsLink` underhand with optional native event) , so IMO it should not break.

Reply:
Indeed, it does not break if `onClick` callback is not specified.